Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM nginx:alpine
+
+ARG BUILD_DATE=""
+ARG BUILD_VERSION="dev"
+
+LABEL maintainer="truebrain@openttd.org"
+LABEL org.label-schema.schema-version="1.0"
+LABEL org.label-schema.build-date=${BUILD_DATE}
+LABEL org.label-schema.version=${BUILD_VERSION}
+
+RUN sed -i 's/access_log/# access_log/g' /etc/nginx/nginx.conf
```

Comment:
```suggestion
RUN sed -i 's/^access_log/# access_log/g' /etc/nginx/nginx.conf
```
?

Reply:
Sadly, no. If I remember correctly, there can be spaces / tabs before this line. And depending on the version, this can change. So your suggestion might break it :(

Reply:
```suggestion
RUN sed -i 's/^\s*access_log/# access_log/g' /etc/nginx/nginx.conf
```
:p